modules/configuring-the-lora-training-job.adoc

```diff
@@ -0,0 +1,249 @@
+endif::[]
+ifdef::upstream[]
+* If Kueue is not installed in your environment, you have configured the Training Operator permissions as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-the-training-operator-permissions-when-not-using-kueue_distributed-workloads[Configuring the Training Operator permissions when not using Kueue].
+endif::[]
+
+* You have access to a model.
+* You have access to data that you can use to train the model.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<username>__ -p __<password>__
+----
+
+. Configure a LoRA training job, as follows:
+.. Create a YAML file named `lora-config.yaml`.
+.. Add the following `ConfigMap` object definition:
++
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: lora-config
+  namespace: kfto
+data:
+  lora_config.json: |
+    {
+      "model_name_or_path": "bigscience/bloom-560m",
+      "training_data_path": "/data/input/twitter_complaints.json",
+      "output_dir": "/data/output/tuning/bloom-twitter",
+      "num_train_epochs": 10.0,
+      "per_device_train_batch_size": 4,
+      "gradient_accumulation_steps": 4,
+      "learning_rate": 1e-05,
+      "response_template": "\n### Label:",
+      "dataset_text_field": "output",
+      "use_flash_attn": false,
+      "lora_r": 8,
+      "lora_alpha": 8,
+      "lora_dropout": 0.1,
+      "bias": "none",
+      "target_modules": ["q_proj", "v_proj"]
+    }
+
+----
+.. Edit the training job metadata as shown in the following table.
++
+.Training job metadata
+[cols="16,84"]
+|===
+|Parameter | Value
+
+|`name`
+|Name of the training job
+
+|`namespace`
+|Name of your project
+|===
+
+.. Edit the parameters of the training job as shown in the following table.
++
+.Training job parameters
+[cols="30,70"]
+|===
+|Parameter | Value
+
+|`model_name_or_path`
+|Name of the pre-trained model
+
+|`training_data_path`
+|Path to the training data that you set in the `training_data.yaml` ConfigMap
+
+|`output_dir`
+|Output directory for the model
+
+|`num_train_epochs`
+|Number of epochs for training; in this example, the training job is set to run 10 times
+
+|`per_device_train_batch_size`
+|Batch size, the number of data set examples to process together; in this example, the training job processes 4 examples at a time
+
+|`gradient_accumulation_steps`
+|Number of gradient accumulation steps
+
+|`learning_rate`
+|Learning rate for the training
+
+|`response_template`
+|Template formatting for the response
+
+|`dataset_text_field`
+|Dataset field for training output, as set in the `training_data.yaml` config map
```

Comment:
In line 114, you use "ConfigMap", but here, in line 135, you use "config map".

Reply:
Thank you, updated to use "config map" for all general references.